plugins/jobs/src/js/components/JobsOverviewList.tsx

```diff
@@ -69,25 +71,56 @@ class JobsOverviewList extends React.Component<
     ) : null;
 
     return (
-      <JobsPage
-        addButton={{
+      <div className="flex-grow">
+        {headline}
+        <FilterBar>
+          <FilterInputText
+            className="flush-bottom"
+            handleFilterChange={this.handleFilterChange}
+            placeholder={i18n._(t`Search`)}
+            searchString={this.props.filter || ""}
+          />
+        </FilterBar>
+        <JobsOverviewTable data={data} />
+      </div>
+    );
+  }
+
+  getEmptyContent() {
+    return (
```

Comment:
NitPick, feel free to ignore: Why don't we make the getXZYs small functional component that lives inside this file? I have seen this quite some times now, but mostly in older code where I could not ask for the reason 😇